yellowbrick/draw.py

```diff
@@ -89,3 +90,74 @@ def manual_legend(g, labels, colors, **legend_kwargs):
     # Return the Legend artist
     return g.legend(handles=handles, **legend_kwargs)
 
+
+def bar_stack(data, ax=None, labels=None, ticks=None, colors=None, 
+              orientation='vertical', colormap=None, **kwargs):
+    """
+    An advanced bar chart plotting utility that can draw bar and stacked bar charts from
+    data, wrapping calls to the specified matplotlib.Axes object. 
+
+    Parameters
+    ----------
+    data : 2D array-like 
+        The data associated with the bar chart where the columns represent each bar
+        and the rows represent each stack in the bar chart. A single bar chart would 
+        be a 2D array with only one row, a bar chart with three stacks per bar would 
+        have a shape of (3, b).  
+
+    ax : matplotlib.Axes, default: None
+        The axes object to draw the barplot on, uses plt.gca() if not specified.
+
+    labels : list of str, default: None
+        The labels for each row in the bar stack, used to create a legend. 
+
+    ticks : list of str, default: None
+        The labels for each bar, added to the x-axis for a vertical plot, or the y-axis
+        for a horizontal plot.  
+
+    colors : array-like, default: None
+        Specify the colors of each bar, each row in the stack, or every segment.
+
+    colormap : string or matplotlib cmap
+        Specify a colormap for each bar, each row in the stack, or every segment.
+
+    kwargs : dict 
+        Additional keyword arguments to pass to ``ax.bar``. 
+    """
+    if ax is None:
+        ax = plt.gca()
+    
+    colors = resolve_colors(n_colors=data.shape[0], 
+                            colormap=colormap, 
+                            colors=colors)    
+    
+    idx = np.arange(data.shape[1])
+    prev = np.zeros(data.shape[1])
+    if orientation == 'vertical':
```

Comment:
could we make this test:

```
orientation = orientation.lower()

if orientation.startswith('v'):

elif orientation.startswith('h'):

else:
    raise YellowbrickValueError("unknown orientation '{}'".format(orientation)
```

This will just make it easier for us super users to pass `bar_plot(orientation='h')` and look super foo!

Reply:
Done!